### ISSUE TITLE
Reify.v invalidates valid syntax

### ISSUE THREAD

**Issue by jesper-bengtson** [gmalecha/mirror-core#100]
After importing MirrorCore.Reify.Reify the following command stops working:

Implicit Arguments id [[A] [B] ...]

Implicit Arguments is deprecated and Arguments should be used in stead, but it is still valid Coq syntax that does not work.

**Comment by gmalecha** [comment]
I agree this is a problem. In the meantime, you should change `Implicit Arguments` into `Arguments` or put a space between the two `[`s.
